pagopa/pn-templates-engine
# PN-13228 - Create template generator script

## Short description
Develop a template generator script that converts MJML files into HTML and generates fully translated HTML templates from them.

## List of changes proposed in this pull request
- Import all assets from `pn-email-templates`
- Implement a generator script with an MJML parser
- Add tests

## How to test
1. Run `npm run build`
2. Verify that the `target` folder contains a subfolder named `generated-templates-assets`, which should include all necessary assets (`fonts`, `images`, `styles`, `templates`)
3. Open and review some of the generated templates to confirm correctness
